manager/controllers/app/fybrikapplication_controller.go

```diff
@@ -798,11 +798,40 @@ func (r *FybrikApplicationReconciler) GetAllModules() (map[string]*fappv1.Fybrik
 		return moduleMap, err
 	}
 	for ind := range moduleList.Items {
+		if !validateModule(&moduleList.Items[ind]) {
```

Comment:
if a module is not valid, should we unregister it? instead of validating again and again.
In anyway, I'd suggest to add a WARN/INFO message about invalid module.

Reply:
Added a warning.